_docs/_latest/concepts/config-validator.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Config Validator
+order: 007
+---
+
+# {{ page.title }}
+
+This page describes how Config Validator works.
+
+---
+
+## **Overview**
+
+The Config Validator Scanner allows you to scan for non-compliant resources in 
+your GCP infrastructure. 
+
+### **How It Works**
+
+- Cloud admins write security and governance constraints (as YAML files) once, 
+and store them within their company’s dedicated Git repo as a central source of 
+truth.
+- Forseti ingests policy constraints written in Rego and uses them as a new 
```

Comment:
(nit) Technically Forseti does not do anything with the constraints, CV ingests the policies.

Reply:
This was taken from the blog post written by Alex Sung. PTAL at #2 

![image](https://user-images.githubusercontent.com/41486977/78376141-e32d5100-759b-11ea-83f1-0d2b63d4f3f9.png)

Let me know if you think sentence on line 22 and blog post needs to be edited.